src/core/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.20)
+
+add_executable(core.cpp core)
```

Comment:
the prototype of this function is `add_executable(<NAME> [SOURCE1] [SOURCE2...])` so the arguments must be passed the other way around